packages/app/src/types.ts

```diff
@@ -1,6 +1,6 @@
 import { z } from 'zod';
 
-import { DashboardSchema, SavedSearchSchema } from './commonTypes';
+import { AlertSchema, DashboardSchema, SavedSearchSchema } from './commonTypes';
```

Comment:
style: we need to add `AlertSchema` to common-utils later. I'm fine to import it here in commonTypes temp

Reply:
done